src/rebuild/srv.c

```diff
@@ -1161,15 +1208,31 @@ rebuild_task_ult(void *arg)
 		D_ERROR(""DF_UUID" (ver=%u) rebuild failed: rc %d\n",
 			DP_UUID(task->dst_pool_uuid), task->dst_map_ver, rc);
 
-		D_GOTO(out, rc);
+		D_GOTO(done, rc);
```

Comment:
here should propagate the rgt status (for example the rgt->rgt_status.rs_errno) to rebuild_iv before do rebuild_iv_update below?
And if this rebuild failed and merged to next rebuild, current test code that wait rebuild's completion and verify data seems possible fail?

Reply:
oh, the leader already to the status (error no), so it does not have to propagate the status to other nodes, and it simply tell other nodes to stop the rebuild by IV (rib_global_done = 1).

oh, this rebuild failed will only happen in REBUILD19 by fault injection. And we will not verify the data after the 1st rebuild failure anyway. Or I miss your question?

Reply:
as discussed just now, it possibly cause lost err when rebuild_status_completed_update in iv refresh, but not a problem for this patch.